gestalt-asset-core/src/main/java/org/terasology/gestalt/assets/management/MapAssetTypeManager.java

```diff
@@ -52,6 +48,17 @@
     private final ListMultimap<Class<? extends Asset>, Class<? extends Asset>> subtypes =
             Multimaps.synchronizedListMultimap(ArrayListMultimap.<Class<? extends Asset>, Class<? extends Asset>>create());
 
+    private static Iterable<Class<?>> getAllSuperTypesBetween(Class<?> upperBound, Class<?> lowerBound) {
```

Comment:
maybe something like this is more understandable: 
```suggestion
    private static Iterable<Class<?>> getAllSuperTypes(Class<?> from, Class<?> to) {
```